src/timer/TimerProvider.jsx

```diff
@@ -53,12 +52,6 @@ const TimerServiceProvider = ({
     },
   ).join(':');
 
-  const pollExam = () => {
```

Comment:
the changes to this file seem to be removing the polling with the backend entirely, is that intended? We still need this to make sure the JS timer stays in sync with the backend